tests/integration-tests/tests/cloudwatch_logging/test_cloudwatch_logging.py

```diff
@@ -584,51 +575,41 @@ def get_config_param_vals(cw_logging_enabled):
         )
     )
     queue_size = int(environ.get("CW_LOGGING_QUEUE_SIZE", 1))
-    return {"enable": str(cw_logging_enabled).lower(), "retention_days": retention_days, "queue_size": queue_size}
-
-
-@pytest.mark.parametrize("cw_logging_enabled", [True, False])
-@pytest.mark.parametrize("cw_logs_persist_after_delete", [True, False])
-@pytest.mark.regions(["us-east-1", "cn-north-1", "us-gov-west-1"])
-@pytest.mark.instances(["t2.micro", "c5.xlarge"])
-def test_cloudwatch_logging(
-    region,
-    scheduler,
-    instance,
-    os,
-    pcluster_config_reader,
-    clusters_factory,
-    cw_logging_enabled,
-    cw_logs_persist_after_delete,
-):
+    return {"enable": "true", "retention_days": retention_days, "queue_size": queue_size}
+
+
+@pytest.mark.regions(["us-east-1"])
+@pytest.mark.oss(["centos7", "ubuntu1804", "alinux2"])
```

Comment:
I'd probably keep testing on all OSes. At most if schedulers logs are OS agnostic we could test only slurm on all OSes and then sge and torque on a single one

Reply:
The reason I did this is because for each scheduler the same log streams should be created for the pairs (centos 6 , centos 7), (ubuntu16, ubuntu18), and (AL1, AL2).

Reply:
I made this change. AL2 is tested with all the schedulers and the other OSes are tested with only slurm.